c_glib/test/test-array.rb

```diff
@@ -116,6 +116,18 @@ def test_to_s
     CONTENT
   end
 
+  def test_validation_success
+    array = build_int32_array([1, 2, 3, 4, 5])
+    assert_equal(true, array.validate)
+  end
+
+  def test_validation_fail
+    array = Arrow::Int8Array.new(-1, Arrow::Buffer.new(""), Arrow::Buffer.new(""), -1)
+    assert_raise(Arrow::Error::Invalid) do
+      array.validate
+    end
```

Comment:
Could you also check error message like https://github.com/apache/arrow/blob/f8f17a58adb6b5fe4fed6c13d7d3b2e64b4a9acd/c_glib/test/test-array.rb#L179-L185 ?

Reply:
Added error message check.